android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
I m really sorry about this change  I don't know why using gradle 2.3.0 it keep giving me errors.  I need to change it to 2.2.3 to make it work.